### PR TITLE
Release v0.0.6

### PR DESCRIPTION
**Pull request by allow-cookies**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+STORAGES_BACKEND=storages.backends.file_system.FileSystemStorage
+STORAGES_PATH=.
+STORAGES_AWS_ACCESS_KEY_ID=AKIACN3948TVUEXAMPLE
+STORAGES_AWS_SECRET_ACCESS_KEY=93hgocns02yt690gjeof95hgkgREXAMPLESECRET
+STORAGES_BUCKET_NAME=your-bucket-name
```
Comment (bplociennik) [inline .env.example]: what about adding Google Cloud examples also? We have it in README but if we have S3 let's add all of it here.

File: README.md [hunk 2]
```diff
@@ -43,4 +44,111 @@ Now, depending on the storage you want to use, follow the instructions below.
...
+> 
+> If the linting process fails or tests wouldn't pass the whole build will fail, so please remember to go through the
+> steps 1-3 and make sure that no problems are being reported to avoid these sad ❌ symbols next to your build.
+
+### ⏩ Before creating a Pull Request
+
+1. Make sure that you've maintained the 100% test coverage of the codebase.
+2. Make sure you've updated the `README.md` file if there are any changes in that may make the `README.md` file obsolete.
+
+### ✅ When creating a Pull Request
```
Comment (bplociennik) [inline README.md]: what about moving those details regarding contribution to a separate file like CONTRIBUTING.md?
+ (+) README will be more readable for people using our package
- (-) contributors have to look at a separate file

**Review by bplociennik — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (allow-cookies) [reply README.md]: Good point. Moved to `CONTRIBUTING.md` file.

File: .env.example (hunk 1 above)
Comment (allow-cookies) [reply .env.example]: Sure, added 🙂

**Pull request merged**